src/Components/MainSection/Notes/Notes.css

```diff
@@ -0,0 +1,56 @@
+.notes-container {
+	padding: 1rem 2rem;
+	height: 85vh;
+}
+.notes-heading {
+	padding: 1rem 0 2rem 0;
+
```

Comment:
you can remove extra spaces and use prettier

Reply:
done